api/proto/teleport/externalaudit/v1/externalaudit_service.proto

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.externalaudit.v1;
+
+import "google/protobuf/empty.proto";
+import "teleport/externalaudit/v1/externalaudit.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/externalaudit/v1;externalauditv1";
+
+// ExternalAuditService provides methods to manage external audit.
+// It's used only in Teleport Cloud for feature called "bring your own audit bucket".
+//
+// There could be multiple external audit resources in cluster, but only single
+// resource can be set as 'Active'.
+// The one which is active is used by auth.
+//
+// Supporting multiple ExternalAudit as resources allow us to to add resource,
+// and test connection before actually enabling it in cluster.
+service ExternalAuditService {
```

Comment:
When are we executing the infra bootstrap?

Reply:
I think it will passed either as a flag to `CreateExternalAudit` or separate RPC endpoint. Maybe separate endpoint makes most sense.

Reply:
So, is it a separate endpoint?

Reply:
yep, i think so, bootstrap can also take a while (up to few seconds - although I haven't measure it yet), so it makes sense to keep is as separate endpoint.

Reply:
I assume we are going to add it to a separate PR

Reply:
yep, exactly, sorry for not being specific